### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bmfmancini**
Fixed install instructions spacing

### CODE REVIEW

**Comment by netniV** [comment]
You need to remove the whitespace at the end of the lines.

**Comment by bmfmancini** [comment]
GRRR I hate MD

**Comment by netniV** [comment]
I like it as you can apply rules to it and get people to conform heh

File: README.md [hunk 1]
```diff
@@ -91,13 +91,9 @@ FOR A PARTICULAR PURPOSE. See the GNU General Public License for more details.
...
 
 3. Installing Cacti on Linux
-
-    3.1.  [Installation Under CentOS 7 - LAMP Stack](Install-Under-CentOS_LAMP.md)
-    
-    3.2.  [Installation Under CentOS 7 - LEMP Stack](Install-Under-CentOS_LEMP.md)
-    
-    3.3. [Installation Under Ubuntu/Debian - LAMP Stack](Installing-Under-Ubuntu-Debian.md)
-
+    - 3.1.  [Installation Under CentOS 7 - LAMP Stack](Install-Under-CentOS_LAMP.md)
```
Comment (netniV) [inline README.md]: Not sure, but I think there are too many spaces here.  The minus should line up with the Installing (a bit like in yaml).

Comment (netniV) [reply README.md]: Also, not sure if there should be a blank line between each element.

Comment (netniV) [reply README.md]: There shouldn't be a mixture of - and numbered lists.

Comment (netniV) [reply README.md]: Now I am wondering what you were attempting to fix?

**Comment by netniV** [comment]
@bmfmancini thanks, I think I understood what you were trying to fix, check out the latest version on develop.

**Pull request closed without merging**